workflows/InsightIDR_Create_Incident_Report_in_ServiceNow/workflow.spec.yaml

```
@@ -0,0 +1,20 @@
+extension: workflow
+product: ["insightconnect"]
+name: InsightIDR_Create_Incident_Report_in_ServiceNow
+title: "InsightIDR Create Incident Report in ServiceNow"
```

This matches `name` in the `.icon` file.

```suggestion
title: "Create Incident Report in ServiceNow"
```